satpy/sentinel1-false-color.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# script by Martin Raspaud\n",
+    "# modified by Mikhail Itkin\n",
+    "\n",
+    "from datetime import datetime\n",
+    "\n",
+    "from satpy import Scene\n",
+    "from satpy import DatasetID\n",
+    "from satpy.utils import debug_on\n",
+    "from trollimage.image import Image\n",
+    "\n",
+    "\n",
+    "def logenh(data):\n",
+    "    res = (np.log10(data / 5000.) - np.log10(0.0223)) /  (1.0 - np.log10(0.0223))\n",
+    "    res[res < 0] = 0\n",
+    "    return res\n",
+    "\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "\n",
+    "    debug_on()\n",
+    "    scn = Scene(\n",
+    "        sensor='sar-c',\n",
+    "        start_time=datetime(2017, 9, 19, 0, 0),\n",
+    "        base_dir=\".\"\n",
+    "    )\n",
+    "\n",
+    "    composite = 'hv'\n",
+    "\n",
+    "    scn.load([DatasetID(name='measurement', polarization='hh'),\n",
+    "              DatasetID(name='measurement', polarization='hv')])\n",
+    "\n",
+    "    mx = 2000.\n",
+    "\n",
+    "    i = scn.datasets.values()[0]\n",
```

Comment:
Would be nice to have explicitly which dataset is used.
What about using eg `i = scn[DatasetId(name='measurement', polarization='hh')]`

Reply:
ok